ghcide/ghcide.cabal

```diff
@@ -30,6 +30,11 @@ flag ghc-patched-unboxed-bytecode
     default:     False
     manual:      True
 
+flag ekg
+    description: Enable EKG monitoring of the build graph and other metrics on port 8999
```

Comment:
The port is configurable, right? Just seems like an odd detail to have here.

Reply:
Putting it here makes it easy to find :)